src/DotNetStatus/JsonData/JsonData.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Fx.Portability
+{
+    public class JsonData
+    {
+        private HttpClient _client;
+        private readonly string _apiPortUrl;
+
+        public JsonData(string apiPortUrl, ProductInformation productInfo)
+        {
+            _client = CreateHttpClient();
```

Comment:
Why are you creating your own HttpClient here? It should be injected in by the DI framework